src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/Cells/DiaryOverviewDayCellModel.swift

```diff
@@ -172,7 +172,11 @@ final class DiaryOverviewDayCellModel {
 	var formattedDate: String {
 		diaryDay.formattedDate
 	}
-	
+
+	var diaryDayTests: [DiaryDayTest] {
+		return diaryDay.tests
+	}
```

Comment:
Maybe I oversee something but why don't we keep `diaryDay.tests`?

Reply:
I only wanted to make the tests internal, because `diaryDay` is a private property of the cellModel